packages/github/README.md

```diff
@@ -55,3 +55,19 @@ const newIssue = await octokit.issues.create({
   body: 'Hello Universe!'
 });
 ```
+
+## Webhook Payload Typescript Definitions
+`@octokit/webhooks` provides complete typescript definitions for the payloads associated with each event. You can cast the provided payload to these types for more type information as needed.
```

Comment:
nit: consider removing unnecessary words. E.g: *The npm module* `@octokit/webhooks` provides ~complete typescript~ *type* definitions for the *response* payloads ~associated with each event~. You can cast the ~provided~ payload to these types for ~more~ *better* type information ~as needed~.